code/game/objects/items/storage/pouch.dm

```diff
@@ -359,6 +358,19 @@
 	fill_type = /obj/item/explosive/grenade/upp
 	fill_number = 4
 
+/obj/item/storage/pouch/detpack
```

Comment:
no, as any pouch it should be selectable by engie AND starts full, and/or be available in surplus vendor and be empty